.github/workflows/autofix.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: autofix.ci
```

Comment:
The workflow name must be `autofix.ci`.
Otherwise, autofix.ci doesn't work.
This is the restriction of autofix.ci.